packages/documents/src/config/routes.ts

```diff
@@ -11,9 +11,20 @@
  */
 import { documentUploadHandler } from '@documents/features/uploadDocument/handler'
 import { vsExportUploaderHandler } from '@documents/features/uploadVSExportFile/handler'
+import { fetchDocumentHandler } from '@documents/features/getDocument/handler'
 
 export const getRoutes = () => {
   const routes = [
+    // get presigned URL
+    {
+      method: 'POST',
+      path: '/presigned-url',
+      handler: fetchDocumentHandler,
+      config: {
+        auth: false,
```

Comment:
Are you sure this is correct? We call this only from gateway right, so this shouldn't be publicly accessible?

Reply:
you are right. This needs to be authorized.